freud/plot.py

```diff
@@ -107,22 +126,57 @@ def line_plot(x, y, title=None, xlabel=None, ylabel=None, ax=None):
     return ax
 
 
+def histogram_plot(values, title=None, xlabel=None, ylabel=None, ax=None):
+    """Helper function to draw a histogram graph.
+
+    .. moduleauthor:: Jin Soo Ihm <jinihm@umich.edu>
+
+    Args:
+        values (list): values of the histogram.
+        title (str): Title of the graph. (Default value = :code:`None`).
+        xlabel (str): Label of x axis. (Default value = :code:`None`).
+        ylabel (str): Label of y axis. (Default value = :code:`None`).
+        ax (:class: matplotlib.axes.Axes): axes object to plot on.
```

Comment:
Probably need backticks around the special class names, here and in other methods.
```rst
:class:`matplotlib.axes.Axes`
```